source/bits/ashiftl.ts

```diff
@@ -0,0 +1,19 @@
+import { Bits } from '../types'
```

Comment:
These files also need to be exported from `source/bits/index.ts`

Reply:
done